privaterelay/settings.py

```diff
@@ -55,9 +55,18 @@
 )
 # maps fxa profile hosts to respective avatar hosts for CSP
 AVATAR_IMG_SRC_MAP = {
-    'https://stable.dev.lcip.org/profile/v1':   'stable.dev.lcip.org',
-    'https://profile.stage.mozaws.net/v1':      'mozillausercontent.com',
-    'https://profile.accounts.firefox.com/v1':  'firefoxusercontent.com',
+    'https://stable.dev.lcip.org/profile/v1':      [
+        'stable.dev.lcip.org',
+        'https://stable.dev.lcip.org/profile/v1/avatar/w',
+    ],
+    'https://profile.stage.mozaws.net/v1':      [
+        'mozillausercontent.com',
+        'https://profile.stage.mozaws.net/v1/avatar/w',
```

Comment:
When I spot-checked on local, the avatar image was `https://profile.stage.mozaws.net/v1/avatar/l` because my stage fxa name starts with `L`. So the path in these directive values needs to be at *most* `/v1/avatar`, but even more flexible (and still safe, assuming we trust ALL images from `profile.stage.mozaws.net`) is to omit the path completely and allow images from the entire origin:
```suggestion
        'https://profile.stage.mozaws.net',
```